sonaranalyzer-dotnet/its/expected/akka.net/Akka.Tests.Shared.Internals-{E636D23C-3432-4AA9-9A5D-5F29D33D3399}-S100.json

```diff
@@ -15,7 +15,7 @@
 },
 {
 "id":  "S100",
-"message":  "Rename method 'expectMsgPF' to match camel case naming rules, consider using 'ExpectMsgPF'.",
+"message":  "Rename method 'expectMsgPF' to match camel case naming rules, consider using 'ExpectMsgPf'.",
```

Comment:
I've tried that, but then there are suboptimal cases - like we would suggest "FSm" instead of "FSM". I'll leave it for now, we can always revise it.